### PR TITLE
DFC-392 add docs and remove console log

### PR DESCRIPTION
**Pull request by di-fabs**
Add docs about trackers

### CODE REVIEW

File: docs/analytics.md [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+This object contains all the methods and logic for running analytics into DI frontend app.
+It creates all the different trackers and starts the cookie management script.
+
+The constructor first creates a new Cookie manager and Page View tracker instances (`dist/docs/pageViewTracker.md`).
+
+If GA4 is not disabled, it will:
+
+- If the user has consented, load the GTM script (using the containerId's passed as parameters)
+- active the Form Response Tracker
```
Comment (SophiaWokoma) [inline docs/analytics.md:36]: activate**

File: docs/analytics.md [hunk 2]
```diff
@@ -0,0 +1,50 @@
...
+
+## How DCMAW wires analytics functionality into its front end
+
+All the javascript files within `src/` are "uglified" into a single, minified `analytics.js` file.
+
+The `analytics.js` file has to be include in the base nunjucks template (e.g: `src/views/common/layout/base.njk`) as a script,
+which makes available all the javascript functions described above on the global window object in the browser.
+
+We then call our `appInit` (`src/index.ts`) function (which is now accessible on the global window
+object) in the same base nunjucks template which initialises analytics and the cookie banner. We pass the `containerId`s and `disableGa4Tracking` as paramters, `disableGa4Tracking`, `disableUaTracking` and `domain` as options;
```
Comment (SophiaWokoma) [inline docs/analytics.md:49]: parameters**

File: docs/analytics.md [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+these values can be inject by your `setLocal` middleware functions which grabs these values from the current environment.
```
Comment (SophiaWokoma) [inline docs/analytics.md:50]: injected**

File: docs/formTrackers.md [hunk 4]
```diff
@@ -0,0 +1,95 @@
...
+'link_path_parts.2': 'undefined',
+'link_path_parts.3': 'undefined',
+'link_path_parts.4': 'undefined',
+'link_path_parts.5': 'undefined'
+}
+
+## Form Error Tracker
+
+If a user has consented to analytics cookies, then the
+`form_error` event is pushed when the user loads a form which contains form error message.
```
Comment (SophiaWokoma) [inline docs/formTrackers.md:46]: contains a **

File: docs/formTrackers.md [hunk 5]
```diff
@@ -0,0 +1,95 @@
...
+"link_path_parts.2": "undefined",
+"link_path_parts.3": "undefined",
+"link_path_parts.4": "undefined",
+"link_path_parts.5": "undefined"
+}
+
+## Form Change Tracker
+
+If a user has consented to analytics cookies, then the
+`form_change` event is pushed when the user loads a form because he/she wants to change the answer.
```
Comment (SophiaWokoma) [inline docs/formTrackers.md:73]: they want to**

File: docs/pageViewTracker.md [hunk 6]
```diff
@@ -0,0 +1,141 @@
...
+});`
+
+## Properties
+
+#### content_id
+
+Content ID is a unique ID for each front end display on a given page. It is detached from the page url and title and can be used to identify a page interacting irrespective of copy updates or multiple front end displays i.e. a page url that has 2 front end displays based on device sniffing. E.g. the download app page in the DCMAW journey.
+100 characters or fewer, lowercase, Guid with hyphens
+
+#### first_publish_at
```
Comment (SophiaWokoma) [inline docs/pageViewTracker.md:48]: first_published_at **

**Review by SophiaWokoma — CHANGES_REQUESTED** [review]
looks good to me , just spotted a few typos :)

File: docs/analytics.md [hunk 7]
```diff
@@ -0,0 +1,50 @@
...
+- active the Form Response Tracker
+  > `dist/docs/formTracker.md`
+- active the Navigation Tracker
+  > `dist/docs/navigationTracker.md`
+
+## How DCMAW wires analytics functionality into its front end
+
+All the javascript files within `src/` are "uglified" into a single, minified `analytics.js` file.
+
+The `analytics.js` file has to be include in the base nunjucks template (e.g: `src/views/common/layout/base.njk`) as a script,
```
Comment (Joanneyoung01) [inline docs/analytics.md:45]: *included

File: docs/analytics.md (hunk 1 above)
Comment (SophiaDWokoma) [inline docs/analytics.md:36]: activate**

File: docs/analytics.md [hunk 8]
```diff
@@ -0,0 +1,50 @@
...
+which makes available all the javascript functions described above on the global window object in the browser.
+
+We then call our `appInit` (`src/index.ts`) function (which is now accessible on the global window
```
Comment (SophiaDWokoma) [inline docs/analytics.md:48]: parameters**

File: docs/analytics.md (hunk 3 above)
Comment (SophiaDWokoma) [inline docs/analytics.md:50]: injected**

File: docs/formTrackers.md (hunk 4 above)
Comment (SophiaDWokoma) [inline docs/formTrackers.md:46]: contains a form error message/(s)**

File: docs/formTrackers.md (hunk 5 above)
Comment (SophiaDWokoma) [inline docs/formTrackers.md:73]: they want to ***

File: docs/pageViewTracker.md (hunk 6 above)
Comment (SophiaDWokoma) [inline docs/pageViewTracker.md:48]: first_published_at **

**Review by SophiaDWokoma — CHANGES_REQUESTED** [review]

**Review by Joanneyoung01 — APPROVED** [review]

**Review by SophiaDWokoma — APPROVED** [review]

**Pull request merged**